fitnessllm_dataplatform/stream/strava/entities/enums.py

```
@@ -0,0 +1,35 @@
+    """Strava specific URLs."""
+
+    AUTH_URL = "https://www.strava.com/oauth/token"
+    ATHLETE_URL = "https://www.strava.com/api/v3/athlete"
+    ACTIVITY_URL = "https://www.strava.com/api/v3/athlete/activities"
+    STREAM_URL = "https://www.strava.com/api/v3/activities/{}/stream"
+
+
+class StravaKeys(Enum):
+    """Strava specific keys."""
+
+    STRAVA_ACCESS_TOKEN = "strava_access_token"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "STRAVA_ACCESS_TOKEN"

(S105)